tests/monitoring/vm_monitoring.go

```diff
@@ -430,20 +287,8 @@ var _ = Describe("[sig-monitoring]VM Monitoring", Serial, decorators.SigMonitori
 		})
 
 		It("[QUARANTINE] should ensure a stress VM has high dirty rate than a stale VM", decorators.Quarantine, func() {
-			createVM := func(vmName string) *v1.VirtualMachine {
-				By(fmt.Sprintf("Creating a VirtualMachine: %s", vmName))
-				vm := libvmi.NewVirtualMachine(libvmifact.NewFedora(libvmi.WithName(vmName)), libvmi.WithRunStrategy(v1.RunStrategyOnce))
-				vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, metav1.CreateOptions{})
-				Expect(err).ToNot(HaveOccurred())
-
-				By("Waiting for the VirtualMachine to be ready")
-				Eventually(matcher.ThisVM(vm)).WithTimeout(100 * time.Second).WithPolling(time.Second).Should(matcher.BeReady())
-
-				return vm
-			}
-
-			staleVM := createVM("stale-vm-" + rand.String(5))
-			stressedVM := createVM("stressed-vm-" + rand.String(5))
+			staleVM := createRunningVM(virtClient, libvmifact.NewFedora(libvmi.WithName("stale-vm-"+rand.String(5))), v1.RunStrategyOnce)
```

Comment:
@avlitman This test is quarantined, I don't think this code should be touched unless you aim to fix the flakiness of this test.